web/src/App.res

```diff
@@ -104,18 +106,29 @@ let make = () => {
 
   let store = Store.use(initIndex)
   let (state, _) = store
+  let (showAbout, setShowAbout) = React.useState(_ => false)
 
   let _topNav = <Nav variant=#Horizontal> {<> </>} </Nav>
-  let topSearch = <PageHeaderTools> <Search.Top store /> </PageHeaderTools>
-  let headerTools = state.index == "" ? React.null : topSearch
+  let headerTools =
+    <PageHeaderTools>
+      <About isOpen=showAbout onClose={() => setShowAbout(_ => false)} />
+      <PageHeaderToolsGroup>
+        <PageHeaderToolsItem>
+          <div onClick={_ => setShowAbout(_ => true)}> <Patternfly.Icons.InfoAlt /> </div>
+        </PageHeaderToolsItem>
+      </PageHeaderToolsGroup>
+    </PageHeaderTools>
   let nav = <MonocleNav active store />
   let sidebar = state.index == "" ? React.null : <PageSidebar nav />
-  let logo = <span onClick={_ => store->Store.changeIndex("")}> <img src="/logo.png" /> </span>
-  let header = <PageHeader logo headerTools />
+  let logo = <span onClick={_ => store->Store.changeIndex("")}> <img src={logoPath} /> </span>
+  let header = <PageHeader showNavToggle={state.index == "" ? false : true} logo headerTools />
   // This sep prevent footer from hidding page content, not pretty but this works!
   let sep = {<> <br /> <br /> <br /> </>}
 
   <Page header sidebar isManagedSidebar={true}>
+    {state.index == ""
+      ? React.null
+      : <PageSection variant=#Dark> <Search.Top store /> </PageSection>}
```

Comment:
We can change the section background color with:

```css
.pf-c-page__main-section.pf-m-dark-200 {
  --pf-c-page__main-section--m-dark-200--BackgroundColor: #212427;
}
```

This is using the vertical nav background, not sure that is any better though :-)

Reply:
Yes I prefer to keep as it is in the Change for now.